dvc/data/stage.py

```diff
@@ -233,7 +236,7 @@ def _make_staging_url(
     return url
 
 
-def _get_staging(odb: "ObjectDB") -> "ObjectDB":
+def _get_staging(odb: "ObjectDB") -> "ReferenceObjectDB":
```

Comment:
Was this causing mypy complaints? Seems like this change is unrelated.

Reply:
Yep: it is required because since I added type hints to `_load_from_state()`